### ISSUE TITLE
Uncaught error:  dependencies.reduce is not a function

### ISSUE THREAD

**Issue by jacksaunders** [auth0/wt-cli#119]
Trying to create a new webtask with the stripe node_module

My Webtask 
```JS
var stripe = require("stripe")

module.exports = function(context, callback) {
	stripe.customers.create({
    api_key: context.secrets.stripeSecretKey,
    email: context.data.email
}, function(err, customer) {
  if(err) {
    callback(null, err)
  } else {
    callback(null, customer)
  }
});
}
```

WT CLI task

```
wt create stripeCreateCustomer.js --name test-stripe_create_customer --secret stripeSecretKey="STRIPE_KEY"
```

Full Error

```
* Hint: A package.json file has been detected adjacent to your webtask. Ensuring that all dependencies from that file are avialable on the platform. This may take a few minutes for new versions of modules so please be patient.
* Hint: If you would like to opt-out from this behaviour, pass in the --ignore-package-json flag.
Uncaught error:  dependencies.reduce is not a function
TypeError: dependencies.reduce is not a function
    at Modules.provision.tap.dependencies (/Users/jacks/.nvm/versions/node/v7.9.0/lib/node_modules/wt-cli/lib/webtaskCreator.js:69:59)
    at Object.tapHandler (/Users/jacks/.nvm/versions/node/v7.9.0/lib/node_modules/wt-cli/node_modules/bluebird/js/main/finally.js:64:23)
    at Object.tryCatcher (/Users/jacks/.nvm/versions/node/v7.9.0/lib/node_modules/wt-cli/node_modules/bluebird/js/main/util.js:26:23)
    at Promise._settlePromiseFromHandler (/Users/jacks/.nvm/versions/node/v7.9.0/lib/node_modules/wt-cli/node_modules/bluebird/js/main/promise.js:510:31)
    at Promise._settlePromiseAt (/Users/jacks/.nvm/versions/node/v7.9.0/lib/node_modules/wt-cli/node_modules/bluebird/js/main/promise.js:584:18)
    at Promise._settlePromises (/Users/jacks/.nvm/versions/node/v7.9.0/lib/node_modules/wt-cli/node_modules/bluebird/js/main/promise.js:700:14)
    at Async._drainQueue (/Users/jacks/.nvm/versions/node/v7.9.0/lib/node_modules/wt-cli/node_modules/bluebird/js/main/async.js:123:16)
    at Async._drainQueues (/Users/jacks/.nvm/versions/node/v7.9.0/lib/node_modules/wt-cli/node_modules/bluebird/js/main/async.js:133:10)
    at Immediate.Async.drainQueues (/Users/jacks/.nvm/versions/node/v7.9.0/lib/node_modules/wt-cli/node_modules/bluebird/js/main/async.js:15:14)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
Please report this at: https://github.com/auth0/wt-cli/issues
```

**Comment by tjanczuk** [comment]
@ggoodman This happens in 5.1.3 when there is a package.json file in the directory *without* any dependencies listed, e.g. 

```
{"name":"foo","version":"1.0.0"}
```

@jacksaunders You can work around this issue by passing `--ignore-package-json` flag to `wt create`.

**Comment by jacksaunders** [comment]
Amazing, worked thanks

**Comment by ggoodman** [comment]
Root cause fixed in e5597bcaf7670bbeb49339f04d31ac882bb70d1d.
